src/sentry/static/sentry/app/components/events/contexts/gpu/getGPUKnownData.tsx

```diff
@@ -1,15 +1,17 @@
-import {KeyValueListData} from 'app/components/events/interfaces/keyValueList/types';
+import KeyValueList from 'app/components/events/interfaces/keyValueList';
 import {getMeta} from 'app/components/events/meta/metaProxy';
 import {defined} from 'app/utils';
 
 import getGpuKnownDataDetails from './getGPUKnownDataDetails';
 import {GPUData, GPUKnownDataType} from './types';
 
+type KeyValueListData = NonNullable<React.ComponentProps<typeof KeyValueList>['data']>;
```

Comment:
You have this type in a few places. Should there be an importable type for this?